service.go

```diff
@@ -97,19 +97,142 @@ func (s *Service) HasTool(category ToolCategory, name string, environment string
 }
 
 func (s *Service) Hydrate(client *Client) error {
-	if err := s.Tags.Hydrate(s.Id, client); err != nil {
-		return err
+	if s.Tags.PageInfo.HasNextPage {
+		variables := &PayloadVariables{}
+		(*variables)["after"] = s.Tags.PageInfo.End
+		_, err := s.GetTags(client, variables)
+		if err != nil {
+			return err
+		}
 	}
-	if err := s.Tools.Hydrate(s.Id, client); err != nil {
-		return err
+
+	if s.Tools.PageInfo.HasNextPage {
+		variables := &PayloadVariables{}
+		(*variables)["after"] = s.Tools.PageInfo.End
+		resp, err := s.GetTools(client, variables)
+		if err != nil {
+			return err
+		}
+		s.Tools = resp
```

Comment:
the tags code doesn't have this.  I cannot remember why or which way was better/more correct

Reply:
Removing it was correct since the Get* functions should populate each field on service object.  Removing them now.